src/MongoDB.Driver/ClientSessionHandle.cs

```diff
@@ -32,6 +35,7 @@ internal sealed class ClientSessionHandle : IClientSessionHandle
         private bool _disposed;
         private readonly ClientSessionOptions _options;
         private readonly IServerSession _serverSession;
+        private readonly IClock _clock;
```

Comment:
Minor nit: keep fields in alphabetical order.

Reply:
Done